### PR TITLE
Release teams message in graph beta endpoint

### PR DESCRIPTION
**Pull request by yiwenwang-ms**

### CODE REVIEW

File: concepts/search-concept-chat-messages.md [hunk 1]
```diff
@@ -0,0 +1,221 @@
+---
```
Comment (isvargasmsft) [inline concepts/search-concept-chat-messages.md:1]: Please add this new concept topic to the conceptual TOC, under Search.

File: concepts/search-concept-chat-messages.md [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+      "query": {
+        "queryString": "contoso from:bob to:alice sent>2022-07-14"
+      },
+      "from": 0,
+      "size": 15,
+      "enableTopResults": true
+    }
+  ]
+}
+```
```
Comment (isvargasmsft) [inline concepts/search-concept-chat-messages.md:196]: Would you be so kind to add the response to this example, please?

File: concepts/search-concept-chat-messages.md [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+| Scope Terms                           | Description   | Example   |
+|:-----------------------------------|:--------------|:--------------------|
+| from | Search only for messages sent by scoped person | from:bob |
+| hasAttachment      | Search only for messages which contains attachment or not | hasAttachment:true |
+| IsRead       | Search only for messages which were read or not | IsRead:true | 
+| IsMentioned        | Search only for messages which mentioned you or not | IsMentioned:true | 
+| mentions      | Search only for messages which mentioned somebody | mentions:497b7a2a9e1a48d780e82965d2fc3a81 (This is user id without '-')| 
+| sent | Search only for messages sent to scoped date range | sent > 2022-07-14 | 
+| to | Search only for messages sent to scoped person, partial supported for the 1 on 1 message | to:bob |
+
```
Comment (isvargasmsft) [inline concepts/search-concept-chat-messages.md:208]: Would you please provide context to this table? Please explain what these scopes are, the value they provide and how to use them. 
In addition, is this the best place in the article for this?

File: concepts/search-concept-chat-messages.md [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+                    "total": 1,
+                    "moreResultsAvailable": false
+                }
+            ]
+        }
+    ]
+}
+```
+
+### Example 3: Search Teams Message with KQL
```
Comment (isvargasmsft) [inline concepts/search-concept-chat-messages.md:170]: What does KQL stand for?

**Review by isvargasmsft — CHANGES_REQUESTED** [review]
Hello @yiwenwang-ms, 

Thank you very much for your work in this PR. I left a few comments. My main concern is finding an optimal way to present the table of supported scope terms. 

Please ping me on the side if you have questions. 

Kind regards, 

Isaac

File: concepts/search-concept-chat-messages.md (hunk 1 above)
Comment (yiwenwang-ms) [reply concepts/search-concept-chat-messages.md:1]: Done

File: concepts/search-concept-chat-messages.md (hunk 4 above)
Comment (yiwenwang-ms) [reply concepts/search-concept-chat-messages.md:170]: Added a reference to the introduction of KQL

File: concepts/search-concept-chat-messages.md (hunk 2 above)
Comment (yiwenwang-ms) [reply concepts/search-concept-chat-messages.md:196]: Done

File: concepts/search-concept-chat-messages.md (hunk 3 above)
Comment (yiwenwang-ms) [reply concepts/search-concept-chat-messages.md]: They are the scope terms that supported for Teams message search in the KQL as the queryString under the Example3.

**Comment by Danielabom** [comment]
@yiwenwang-ms Hi Yiwen, I noticed you added the label "review feedback addressed" today. I checked @isvargasmsft comments and your changes, and everything seems to be in place. I'll ask Isaac to run a final review as soon as possible. Isaac was kind enough to take over this PR while I was OOF so I need his final approval.

**Review by isvargasmsft — APPROVED** [review]

**Pull request merged**